schemas/compat-data.schema.json

```
@@ -91,6 +91,26 @@
       ]
     },
 
+    "matches_block": {
+      "type": "object",
+      "maxProperties": 1,
+      "minProperties": 1,
+      "properties": {
+        "keywords": {
+          "type": "array",
+          "minItems": 1,
+          "uniqueItems": true
```

The schema should probably also specify that keywords are expected to be strings.
```suggestion
          "uniqueItems": true,
          "items": {
            "type": "string"
          }
```